src/pages/edit-profile/EditProfile.jsx

```diff
@@ -72,7 +72,7 @@ const EditProfile = () => {
       })
       .catch((e) => {
         setUploading(false);
-        setToast({ message: e.data.description[0], color: "danger" });
+        setToast({ message: e.response.data.username[0], color: "danger" });
```

Comment:
* Why are we reading from `e.response.data` here while in  `Onboarding.jsx` line 59, we are reading `e.data`?
* Why are we reading from username field for edit profile, but description field for edit username?

Reply:
Yup fixed this